docs/headless-chrome.md

```diff
@@ -1,28 +1,62 @@
-# Running headless Chrome for Lighthouse
+# Running Lighthouse using headless Chrome
 
-The headless_shell still has a few bugs to work out. Until then, Chrome + xvfb is a stable solution.
```

Comment:
I think we should keep the reference and instructions to xvfb until we're confident in headless parity, maybe just split into 2 sections and add a caveat to the headless instructions with a link to the section lower down on xvfb?